test/test_pebble.py

```diff
@@ -4030,7 +4030,7 @@ def test_local_identity_from_dict(self):
 
     def test_local_identity_from_dict_with_access_enum(self):
         identity = pebble.Identity.from_dict({
-            'access': pebble.IdentityAccess.ADMIN,
+            'access': pebble.IdentityAccess.ADMIN,  # type: ignore
```

Comment:
This is of potential concern to our users.

If we want them to call `Identity.from_dict` both with strings and enum members and have that type check strictly, we may have to do some extra work.

For now, I've decided not to adjust our public types, not within the scope of this PR anyway.

Reply:
It's not possible to add type annotations to enum members.

+1 on making the user-facing change in a separate PR, with pros and cons discussion on that separate PR.